packages/origin-247-certificate/src/onchain-certificate/certificate-operations/claim/batch-claim-certificates.handler.ts

```diff
@@ -0,0 +1,76 @@
+import { InjectRepository } from '@nestjs/typeorm';
+import { Repository } from 'typeorm';
+import { Certificate, CertificateBatchOperations } from '@energyweb/issuer';
+import { BigNumber, ContractTransaction } from 'ethers';
+import {
+    BadRequestException,
+    ForbiddenException,
+    HttpException,
+    HttpStatus,
+    NotFoundException
+} from '@nestjs/common';
+import { BlockchainPropertiesService } from '../../blockchain-properties.service';
+import { IBatchClaimCommand } from '../../../types';
+import { BatchCertificateClaim } from '@energyweb/issuer/dist/js/src/blockchain-facade/CertificateBatchOperations';
+
+export class BatchClaimCertificatesHandler {
```

Comment:
What is the purpose of these handlers (this and others)? Everything is already validated in offchain layer, isn't it?